<launch>
	<node name="peripheral_arduino_in" pkg="receiver" type="peripheral_arduino_in.py" />

	<node name="safety_motor" pkg="receiver" type="safety_motor.py" />
 
	<node name="status_control" pkg="receiver" type="status_control.py" />
		
	<node name="mode_select" pkg="receiver" type="mode_select.py" />

	

</launch>
